media-gfx/blender/blender-2.79b.recipe

```diff
@@ -33,8 +33,8 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	numpy_python3
-	requests_python3
+	numpy${secondaryArchSuffix}_python36
+	requests_python36
```

Comment:
Does it depend on python 3.6 (or is it possible to switch to python 3.7)?

Reply:
It was bumped to 3.7 earlier, but all the UI text dissapeard, X512 downgraded the recipe to 3.6, which is working well. For 3.7 we need Blender 2.8, what doesn't works yet on Haiku, so we are locked to Python 3.6 here.